### ISSUE TITLE
Carla doesn't start with python-pyqt5 5.15.8

### ISSUE THREAD

**Issue by SnipeXandrej** [falkTX/Carla#1735]
Carla with python-pyqt5 5.15.8 doesn't start and immediately prints a traceback in the terminal, but Carla does start after downgrading python-pyqt5 to 5.15.7

This is on Arch Linux.

```
snipex@Zephyrus ~                                                                                                                                                                                                                                                  [14:51:03]
> $ carla
Traceback (most recent call last):
  File "/usr/share/carla/carla", line 22, in <module>
    from carla_host import *
  File "/usr/share/carla/carla_host.py", line 53, in <module>
    import ui_carla_host
  File "/usr/share/carla/ui_carla_host.py", line 847, in <module>
    from widgets.draggablegraphicsview import DraggableGraphicsView
  File "/usr/share/carla/widgets/draggablegraphicsview.py", line 30, in <module>
    from carla_shared import MACOS, CustomMessageBox, gCarla
  File "/usr/share/carla/carla_shared.py", line 41, in <module>
    from PyQt5.Qt import PYQT_VERSION_STR
ImportError: cannot import name 'PYQT_VERSION_STR' from 'PyQt5.Qt' (/usr/lib/python3.10/site-packages/PyQt5/Qt.abi3.so)
```

**Comment by falkTX** [comment]
Should be fixed in 8c662a990f33e278a738dcc4f5b43c086c12166b, you can use the hotfix-2.5 branch for testing.
Feedback is much appreciated, I dont personally have a way to test this myself for now

**Comment by SnipeXandrej** [comment]
That commit does indeed fix the issue.
Btw there seems to be a regression in sip 6.7.6 that's causing these issues. So as a workaround Arch 2 hours ago pushed an update to the pyqt5 package that basically recompiled pyqt5 with the older sip 6.7.5 instead of sip 6.7.6

https://bugs.archlinux.org/task/77315
